README.md

```diff
@@ -1,3 +1,22 @@
 # node-intuit
 
-`npm` port of https://github.com/cloocher/aggcat
+WIP `npm` port of https://github.com/cloocher/aggcat
+
+### Basic Usage
+
+```
+config =
+  issuerId: "issuer-id"
+  consumerKey: "consumer-key"
+  consumerSecret: "consumer-secret"
+  userId: "1"
+  key: fs.readFileSync path.join(__dirname, "app.key"), "utf8"
```

Comment:
so configuring keys and such is ok, but how do you actually make calls to make requests to `getInstitutions`.